src/plugins/unified_search/server/autocomplete/value_suggestions_route.ts

```diff
@@ -25,6 +25,13 @@ export function registerValueSuggestionsRoute(router: IRouter, config$: Observab
     .addVersion(
       {
         version: '1',
+        security: {
+          authz: {
+            enabled: false,
+            reason:
+              'This route is opted out from authorization because uses the current user authorizations.',
```

Comment:
> [!TIP]
> You can also lift up the security config to the top level definition, this way you won't need to repeat the same justification for authz once you add another version of the API in the future.
```
 router.versioned
    .post({
      path: '/internal/kibana/suggestions/values/{index}',
      access: 'internal',
      security: {
          authz: {
            enabled: false,
            reason:
              'This route is opted out from authorization because uses the current user authorizations.',
           }
        }
    })
   ```